### ISSUE TITLE
Could you let this function public, So I can custom my processor

### ISSUE THREAD

**Issue by harryzjm** [Yeatse/KingfisherWebP#30]
```swift  
extension KingfisherWrapper where Base: Image {
    static func image(webpData: Data, scale: CGFloat, onlyFirstFrame: Bool) -> Image? 
```

**Comment by Yeatse** [comment]
Sure.

**Comment by Yeatse** [comment]
Should be available in 0.7.0

**Comment by harryzjm** [comment]
Thanks

: ]

On Sep 30, 2019, at 6:33 PM, Yang Chao <notifications@github.com> wrote:

﻿

Should be available in 0.7.0
